CMakeLists.txt

```diff
@@ -2,42 +2,31 @@ cmake_minimum_required(VERSION 3.13.0 FATAL_ERROR)
 
 # There is a bug in 3.19.0 that causes .S files to be treated like C files
 if(CMAKE_VERSION VERSION_EQUAL "3.19.0")
-    message(FATAL_ERROR "Dyninst cannot use CMake version 3.19.0")
+  message(FATAL_ERROR "Dyninst cannot use CMake version 3.19.0")
 endif()
 
 set(DYNINST_MAJOR_VERSION 12)
 set(DYNINST_MINOR_VERSION 2)
 set(DYNINST_PATCH_VERSION 1)
 
-set(SOVERSION "${DYNINST_MAJOR_VERSION}.${DYNINST_MINOR_VERSION}")
-set(LIBVERSION "${SOVERSION}.${DYNINST_PATCH_VERSION}")
-set(DYNINST_VERSION "${LIBVERSION}")
+set(DYNINST_SOVERSION "${DYNINST_MAJOR_VERSION}.${DYNINST_MINOR_VERSION}")
+set(DYNINST_LIBVERSION "${DYNINST_SOVERSION}.${DYNINST_PATCH_VERSION}")
```

Comment:
I just blindly updated them, but they can be merged.

Reply:
I would say that dyninst_libversion should never be dyninst_version. Really, they should be independent and follow: https://www.gnu.org/software/libtool/manual/html_node/Libtool-versioning.html

Reply:
@woodard We've internally had this conversation in the past. We decided at that time to leave the SONAME as major.minor because we didn't want to maintain two sets of version numbers. Dyninst isn't alone in doing this, but I agree it doesn't follow the libtools guidelines. @kupsch @bigtrak Maybe we should revisit this for the 13.0 release?